### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by bl-ue**
As usual, when I like a project, I use [codespell](https://github.com/codespell-project/codespell)﻿ to clean up typos in it.

This PR also trims whitespace at the end of lines commonly left by editors, as I have VS Code configured to do that (I hope you don't mind @sbrl!)

### CODE REVIEW

File: Chat-Command-Reference.md
Comment (bl-ue) [inline Chat-Command-Reference.md:82]: Looks like maybe this line was never finished :P

Comment (sbrl) [reply Chat-Command-Reference.md:82]: Oops! I must have been distrated by something in the middle of writing that. I've made a commit to fix that.

File: worldeditadditions/utils/node_identification.lua
Comment (sbrl) [inline worldeditadditions/utils/node_identification.lua:15]: I'm sorry, but I can't merge this if the whitespace on these lines is deleted. Can you restore the whitespace here and on similar lines please?

It's rather annoying in my code editor when I want to edit that line.

**Review by sbrl — CHANGES_REQUESTED** [review]
Hey there, @bl-ue! Didn't know you were a Minetest fan lol!

Just got a comment here about whitespace.

File: worldeditadditions/utils/node_identification.lua
Comment (bl-ue) [reply worldeditadditions/utils/node_identification.lua:15]: Really! No problem, usually that's not desired.  But I'll fix it 👍🏻

Comment (sbrl) [reply worldeditadditions/utils/node_identification.lua]: Thanks :D

Yeah, I'm somewhat unusual lol

**Review by sbrl — APPROVED** [review]
Thanks, @bl-ue :D

**Pull request merged**
